src/HModel.cpp

```diff
@@ -2,7 +2,14 @@
 #include "HConst.h"
 #include "HTimer.h"
 #include "HPresolve.h"
+
+
+#ifdef Boost_FOUND
+#include "HMpsFF.h"
+#else
 #include "HMPSIO.h"
+#endif
```

Comment:
I suppose it would be nicer to integrate the entire code of HMPSFF.h into HMPSIO.h and do the `#ifdef Boost_FOUND` distinction in there.